src/unix/bsd/freebsdlike/freebsd/freebsd11/mod.rs

```diff
@@ -31,99 +31,182 @@ s! {
     }
 
     pub struct kinfo_proc {
+        /// Size of this structure.
         pub ki_structsize: ::c_int,
+        /// Reserved: layout identifier.
         pub ki_layout: ::c_int,
+        /// Address of command arguments.
         pub ki_args: *mut ::pargs,
         // This is normally "struct proc".
+        /// Address of proc.
         pub ki_paddr: *mut ::c_void,
         // This is normally "struct user".
+        /// Kernel virtual address of u-area.
         pub ki_addr: *mut ::c_void,
         // This is normally "struct vnode".
+        /// Pointer to trace file.
         pub ki_tracep: *mut ::c_void,
         // This is normally "struct vnode".
+        /// Pointer to executable file.
         pub ki_textvp: *mut ::c_void,
         // This is normally "struct filedesc".
+        /// Pointer to open file info.
         pub ki_fd: *mut ::c_void,
         // This is normally "struct vmspace".
+        /// Pointer to kernel vmspace struct.
         pub ki_vmspace: *mut ::c_void,
+        /// Sleep address.
         pub ki_wchan: *mut ::c_void,
+        /// Process identifier.
         pub ki_pid: ::pid_t,
+        /// Parent process ID.
         pub ki_ppid: ::pid_t,
+        /// Process group ID.
         pub ki_pgid: ::pid_t,
+        /// tty process group ID.
         pub ki_tpgid: ::pid_t,
+        /// Process session ID.
         pub ki_sid: ::pid_t,
+        /// Terminal session ID.
         pub ki_tsid: ::pid_t,
+        /// Job control counter.
         pub ki_jobc: ::c_short,
+        /// Unused (just here for alignment).
         pub ki_spare_short1: ::c_short,
+        /// Controlling tty dev.
         pub ki_tdev: ::dev_t,
+        /// Signals arrived but not delivered.
         pub ki_siglist: ::sigset_t,
+        /// Current signal mask.
         pub ki_sigmask: ::sigset_t,
+        /// Signals being ignored.
         pub ki_sigignore: ::sigset_t,
+        /// Signals being caught by user.
         pub ki_sigcatch: ::sigset_t,
+        /// Effective user ID.
         pub ki_uid: ::uid_t,
+        /// Real user ID.
         pub ki_ruid: ::uid_t,
+        /// Saved effective user ID.
         pub ki_svuid: ::uid_t,
+        /// Real group ID.
         pub ki_rgid: ::gid_t,
+        /// Saved effective group ID.
         pub ki_svgid: ::gid_t,
+        /// Number of groups.
         pub ki_ngroups: ::c_short,
+        /// Unused (just here for alignment).
         pub ki_spare_short2: ::c_short,
+        /// Groups.
         pub ki_groups: [::gid_t; ::KI_NGROUPS],
+        /// Virtual size.
         pub ki_size: ::vm_size_t,
+        /// Current resident set size in pages.
         pub ki_rssize: ::segsz_t,
+        /// Resident set size before last swap.
         pub ki_swrss: ::segsz_t,
+        /// Text size (pages) XXX.
         pub ki_tsize: ::segsz_t,
+        /// Data size (pages) XXX.
```

Comment:
What do these "XXX" markers mean here? Noticed them while doing a review for cargo-crev, got curious :) They're also present in all other versions of this struct.

Reply:
I have absolutely no idea. I tool the comment as is.